Basic_Concepts/Callback/README.md

```diff
@@ -18,7 +18,7 @@ The basic syntax of Jquery is:
 
 which  
 
- - The sign $ defines that we will work will work with jQuery.   
+ - The sign $ defines that we will work with jQuery.   
```

Comment:
The $ sign defines that we will work with selectors in jQuery. It is known as the factory function.